lib/src/error_group.dart

```diff
@@ -188,7 +188,7 @@ class _ErrorGroupFuture<T> implements Future<T> {
     _completer.future.catchError((_) {});
   }
 
-  Future/*<S>*/ then/*<S>*/(/*=S*/ onValue(/*=T*/ value), {Function onError}) {
+  Future/*<S>*/ then/*<S>*/(onValue(/*=T*/ value), {Function onError}) {
```

Comment:
I updated the signature to match Future in the SDK – for which I was given a strong-mode error.

So this is a known delta?